### PR TITLE
Author block backend

### PR DESCRIPTION
**Pull request by rastislav-chynoransky**
https://jira.sng.sk/browse/NSNG-73

### CODE REVIEW

File: app/Providers/AppServiceProvider.php [hunk 1]
```diff
@@ -23,6 +24,10 @@ public function register()
      */
     public function boot()
     {
-        //
+        Http::macro('webumenia', function () {
```
Comment (igor-kamil) [inline app/Providers/AppServiceProvider.php:27]: ❤️

File: routes/api.php [hunk 2]
```diff
@@ -24,26 +24,24 @@
...
+    $response = Http::webumenia()->get("/v2/items/$id");
+    $item = $response->object()->data;
     if (!empty($code->description)) {
         $item->description = nl2br($code->description);
     }
-    $item->code = $code->code ?? null;
-    $item->offset_top = $code->offset_top ?? 0;
+    $item->code = $code->code;
+    $item->offset_top = $code->offset_top;
+    $item->author_description = $code->author_description;
```
Comment (igor-kamil) [inline routes/api.php:36]: please keep this fallbacks... it is (for now) supposed to work even if you have stored in your collection the artwork, which is removed from `codes` table
that's the reason for fallbacks
you can easily test it with any artwork id - e.g. https://kod.sng.sk/api/items/SVK:PGU.K_1253
suggestion:

```
$item->code = $code->code ?? null;
$item->offset_top = $code->offset_top ?? 0;
$item->author_description = $code->author_description ?? null;
```

**Review by igor-kamil — COMMENTED** [review]
very nice. works smoothly. just keep fallback values in api endpoint - to make it works for any artwork ID

File: routes/api.php (hunk 2 above)
Comment (rastislav-chynoransky) [reply routes/api.php:36]: Thanks, it wasn't clear to me. Then I think the model should be instantiated with some reasonable defaults instead:
`$code = Code::where('item_id', $id)->first() ?? Code::factory()->make([]);`

File: routes/api.php [hunk 3]
```diff
@@ -25,7 +25,7 @@
 });
 
 Route::get('/items/{id}', function (string $id) {
-    $code = Code::where('item_id', $id)->first();
+    $code = Code::where('item_id', $id)->first() ?: Code::factory()->make();
```
Comment (igor-kamil) [inline routes/api.php:28]: 👌

**Review by igor-kamil — APPROVED** [review]
🚀

**Pull request merged**